projects/cdk/utils/svg/svg-linear-gradient-processor.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * @note(splincode):
+ * Any ‘linearGradient’ attributes which are defined on the referenced
+ * element which are not defined on this element are inherited by this element.
+ * If this element has no defined gradient stops, and the referenced element does
+ * (possibly due to its own ‘xlink:href’ attribute), then this element inherits
+ * the gradient stop from the referenced element. Inheritance can be indirect
+ * to an arbitrary level; thus, if the referenced element inherits attribute
+ * or gradient stops due to its own ‘xlink:href’ attribute, then the current
+ * element can inherit those attributes or gradient stops.
+ *
+ * Documentation: https://www.w3.org/TR/SVG11/pservers.html
+ *
+ */
+export function svgLinearGradientProcessor(
+    svg: string,
+    customSalt?: number | string,
+): string {
+    const findAllIds = (svg.match(/url\(#(\w\w+)/g) || []).filter(onlyUnique);
```

Comment:
You can use `new Set`

Reply:
And `find` in the name doesn't make sense :)

Reply:
fixed